Sources/CoreAardvark/Logging/ARKLogDistributor+SwiftAdditions.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+extension ARKLogDistributor {
+
+    /// Creates a log message and distributes it to the log observers.
```

Comment:
Should we note here that this is a thin convenience wrapper for Swift?

Reply:
Eh, I don't think it's important to know that this is a wrapper. In the future it could easily be the opposite (the Objective-C versions are a wrapper around the Swift version) and the API usage wouldn't change.

Reply:
The reason I ask for a comment saying this is a wrapper is because both this method and the other will show up in autocomplete, and it won't necessarily be clear to a consumer reading the doc comments on each autocomplete option which method they should use.

Reply:
Ahh good point. Added in 408d516.